docs/faq-2-user.txt

```diff
@@ -213,3 +213,18 @@
     this increases the size of the default ClientHello message to more than
     255 bytes in length. Some software cannot handle this and hangs.
 
+*   Some secure servers emit an infinite loop of HTTP headers with an OpenSSL
+    1.1.0 client, is this a bug?
+
+    OpenSSL 1.1.0 introduced support for several new TLS extensions, including
+    encrypt-then-mac and extended-master-secret, both of which provide
+    significant security improvements.  Unfortunately, some deployed TLS
+    servers are fatally buggy and do not implement extensibility in a
```

Comment:
I‘d remove „are fatally buggy and“. It sounds very biased.

Reply:
Well, breaking on unknown extensions is a quite fatal bug. Maybe use "are severally broken"?

Reply:
+1 from me for @t8m's suggestion, it sounds less colloquial and more objective.